<ui version="4.0" >
 <class>EquationTab</class>
 <widget class="QWidget" name="EquationTab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>564</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>-1</number>
   </property>
   <property name="verticalSpacing" >
    <number>-1</number>
   </property>
   <item row="2" column="0" >
    <widget class="Kst::CurvePlacement" native="1" name="_curvePlacement" />
   </item>
   <item row="1" column="0" >
    <widget class="Kst::CurveAppearance" native="1" name="_curveAppearance" />
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="GroupBox14" >
     <property name="title" >
      <string>Equation</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="TextLabel1_2_3_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Operators:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>Operators</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="Operators" >
        <property name="whatsThis" >
         <string>Select an operator from this list to insert it into the equation.  All valid operators are listed here.  </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="TextLabel1_2_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Vectors:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>_vectors</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="Kst::VectorSelector" native="1" name="_vectors" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="TextLabel1_2_2_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Scalars:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>_scalars</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="Kst::ScalarSelector" native="1" name="_scalars" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="TextLabel1_11" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Equation:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>_equation</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="_equation" >
        <property name="whatsThis" >
         <string>Enter the equation to be evaluated here.  Use 'x' as the independent variable.  Vectors an operators are selected from the above lists.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>X Vector:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="Kst::VectorSelector" native="1" name="_xVectors" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2" >
       <widget class="QCheckBox" name="_doInterpolation" >
        <property name="text" >
         <string>Interpolate to highest resolution vector</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Kst::VectorSelector</class>
   <extends>QWidget</extends>
   <header>vectorselector.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::CurvePlacement</class>
   <extends>QWidget</extends>
   <header>curveplacement.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::CurveAppearance</class>
   <extends>QWidget</extends>
   <header>curveappearance.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::ScalarSelector</class>
   <extends>QWidget</extends>
   <header>scalarselector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Operators</tabstop>
  <tabstop>_vectors</tabstop>
  <tabstop>_scalars</tabstop>
  <tabstop>_equation</tabstop>
  <tabstop>_xVectors</tabstop>
  <tabstop>_doInterpolation</tabstop>
  <tabstop>_curveAppearance</tabstop>
  <tabstop>_curvePlacement</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Operators</sender>
   <signal>activated(QString)</signal>
   <receiver>_equation</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
